app/client/views/application/application.html

```diff
@@ -5,121 +5,123 @@
 
   <div class="ui stackable page grid">
     <div class="column">
-      <div class="ui form"
+      <form class="ui form"
             ng-class="{'loading': loading}">
 
         <div class="divided title">Basic Information</div>
 
         <fieldset ng-disabled="regIsClosed">
 
-        <div class="field">
```

Comment:
this was just one level of indentation off so i pushed it inone, but the diff is insane...

Reply:
AHHHHHH